swatch-system-conduit/deploy/clowdapp.yaml

```diff
@@ -98,11 +98,9 @@ parameters:
     value: '3'
   - name: HOST_LAST_SYNC_THRESHOLD
     value: '30h'
-  - name: RHSM_API_LIMIT_FOR_PERIOD
-    value: '150'
-  - name: RHSM_API_LIMIT_REFRESH_PERIOD
-    value: 100ms
-  - name: RHSM_API_LIMIT_TIMEOUT_DURATION
+  - name: RHSM_API_MAX_CONCURRENT_CALLS
+    value: '5'
```

Comment:
I think this configuration is too restrictive (only 5 concurrent calls with a timeout of 30m). Looking forward to what you think about it and suggestions.

Reply:
We run 3 replicas and each has a single kafka consumer, so I don't any issue with 5. I'm more interested to see whether this is sufficient to limit API calls enough to keep from overwhelming the RHSM API endpoint.